routes/warehouses-route.js

```diff
@@ -22,4 +22,6 @@ router.route("/:id/inventory").get(warehouseController.inventory);
 
 router.post("/", warehouseValidationRules, warehouseController.addWarehouse);
 
+router.route("/:id").put(warehouseController.update);
```

Comment:
You can chain the put function at the end of the post function
` router.post().put();`